_examples/dart_full_stack/Taskfile.yml

```diff
@@ -0,0 +1,64 @@
+version: "2"
```

Comment:
Would you be cool to move this dart_full_stack example to another repo such as "flutter-webrpc-example", even under your own repo and we can link to it from README? 

as well, inside of webrpc repo, under _examples/ we could make a "dart-app" which includes a simple client and server example, ideally something very minimal

Reply:
I don't have a problem with that. I'm not sure how much more minimal the example could be though. I think most of the complexity is inherent in even the smallest of flutter apps to be honest?. Do you mean fewer methods in the service?

Reply:
or would you like me to forgo using flutter altogether?

Reply:
yes, for the example in this repo specifically, the examples should be as minimal/small as possible.

Reply:
remove flutter, yes

Reply:
but I still think the flutter example is great and would be a nice reference in another example repo.. up to you for that of course

Reply:
No problem, I will write a quick example over the weekend and move the current one to another repo.

Reply:
Hey @pkieltyka , sorry it took me so long to update the examples for this. I've been slammed at work. Anyhow, I have moved the flutter examples to another repo and created simpler example for dart usage. Please let me know if there's anything else needed.